lighthouse-task.ts

```diff
@@ -1,8 +1,7 @@
-const fs = require('fs');
-const path = require('path');
-// Const {exec} = require('child_process');
-const parse = require('csv-parse/lib/sync');
-const { reportsForRows } = require('./lighthouse');
+import fs from 'fs';
+import path from 'path';
+import parse from 'csv-parse/lib/sync';
+import { reportsForRows } from './lighthouse';
```

Comment:
The `reportsForRows` function doesn't exist, so this task won't work. Is this task still used @emersonthis?

Reply:
@calebeby That task file is an artifact from when the project was structured as three discrete scripts. At this point we should probably remove this task file.